app/routes.js

```diff
@@ -0,0 +1,10 @@
+// const controller = require('./controllers/controller');
+const { healthCheck } = require('./controllers/healthCheck');
+const { post } = require('./controllers/weets');
+
+exports.init = app => {
+  app.get('/health', healthCheck);
+  app.post('/weets', [], post);
+  // app.put('/endpoint/put/path', [], controller.methodPUT);
+  // app.post('/endpoint/post/path', [], controller.methodPOST);
+};
```

Comment:
Here you could eliminate the endpoint of the weets, you could also eliminate the commented lines since they are only for example.